src/paint.js

```diff
@@ -154,7 +154,9 @@ export default function paint($element, layout, component) {
 
   title.append('text')
     .attr('class', 'title')
-    .text(function (d) { return d.title; });
+    .text(function (d) {
+      if(d.title === undefined) return 'undefined';
```

Comment:
in the given application the one without it has '-' in it, so this won't trigger. I am guessing this is a safety null-check then. Should we cover null or empty string cases as well?

Reply:
'-' indicates a Nan value,, titles with "undefined" is a different thing.
both has been handled seperately